src/Usabilla/API/Description/UsabillaDescription.php

```diff
@@ -54,6 +54,23 @@ public function __construct()
                     ],
                 ],
 
+                'GetAppCampaigns' => [
+                    'httpMethod' => 'GET',
+                    'uri' => '/live/apps/campaign',
+                    'responseModel' => 'AppCampaignItems',
+                ],
+
+                'GetAppCampaignResultItems' => [
```

Comment:
Please let's keep the naming convention, should be `GetAppCampaignResults` (since the one for feedback items is `GetWebsiteCampaignResults`)